docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -58,7 +58,10 @@ Settings are managed using the NuGet CLI [config command](../reference/cli-refer
 Windows:
 
 ```cli
-# Set repositoryPath in the user-level config file
+# Set globalPackagesFolder in the user-level config file
+nuget config -set globalPackagesFolder=c:\packages
```

Comment:
I think these examples should be changed to `dotnet nuget config` syntax, so it can be copy-pasted by more customers.